sections/06_discussion.md

```diff
@@ -0,0 +1,54 @@
+## Discussion
+
+*This section provides meta-commentary that spans the Categorize, Study, and
+Treat subject areas.  The candidate sub-sections below are initial ideas that
+can be further pruned.*
+
+### Evaluation
+
+*What are the challenges in evaluating deep learning models that are specific to
+this domain?  This can include a discussion of ROC versus precision-recall
+curves for the imbalanced classes often encountered in biomedical datasets.
+It could also mention alternative metrics that are used in specific sub-areas
+such as enrichment factors in virtual screening.  A lack of true gold standard
+data for some problems complicates both training and evaluation.  Confidence-
+weighted labels are valuable when available.*
+
+### Interpretation
+
+*Most of our examples pertain to the Study papers.  Does this discussion
+belong there or can we generalize it and keep it here?  Specific points would
+include the dangers of over-interpreting hidden units, pros/cons of specific
+techniques (see issues), and recommendations.  Some other reviews have addressed
+this in part as well.*
+
+### Data limitations
+
+*Related to evaluation, are there data quality issues in genomic, clinical, and
+other data that make this domain particularly challenging?  Are these worse than
+what is faced in other non-biomedical domains?*
+
+*Many applications have used relatively small training datasets.  We might
+discuss workarounds (e.g. semi-synthetic data, splitting instances, etc.) and
+how this could impact future progress.  Might this be why some studies have
+resorted to feature engineering instead of learning representations from low-
+level features?  Is there still work to be done in finding the right low-level
+features in some problems?*
+
+### Hardware limitations and scaling
+
+*Several papers have stated that memory or other hardware limitations
+artificially restricted the number of training instances, model inputs/outputs,
+hidden layers, etc.  Is this a general problem worth discussing or will it be
+solved naturally as hardware improves and/or groups move to distributed deep
+learning frameworks?  Does hardware limit what types of problems are accessible
+to the average computational group, and if so, will that limit future progress?
+For instance, some hyperparameter search strategies are not feasible for a lab
+with only a couple GPUs.*
+
+*Some of this is also outlined in the Categorize section.  We can decide where
+it best fits.*
+
+### Code, data, and model sharing
+
+*Anything to be said?*
```

Comment:
I think it would be useful to discuss reproducibility here - even though the importance of reproducibility is not specific for our topic. 

However, when deep models are being built the hyperparameter space is so large and data is so heterogeneous we may need to think about how reproducibility in deep learning in biomedical science, and the impact it has on improving healthcare, is different than reproducibility in the broader computational biology context. To slightly rephrase this question: "what might a clinician, or a policy maker, need to see in order to influence treatment decisions". 

Might this be a similar situation as a requirement for reproducible GWAS signals?